packages/launcher-component/src/core/stuff.tsx

```diff
@@ -36,7 +36,7 @@ export function SpecialValue(props: { children: string }) {
 export function AlertError(props: { error: any }) {
   return (
     <Alert variant={AlertVariant.danger} title="Something weird happened:" aria-label="error-in-hub-n-spoke" style={{margin: '40px'}}>
-      {props.error.toString()}
+      {props.error.message}
```

Comment:
You have no guaranty that the error will have `props.error.message`. I would prefer is you the error sent by the client would implement `toString()` with the message.

Reply:
see my other comment there will be a message plus the extra information as to why the error happend